Tools/ApplicationDebugger/jacobi/README.md

```diff
@@ -0,0 +1,340 @@
+# `jacobi` Sample
```

Comment:
I made several minor edits, please take a look and make sure you are in agreement with changes

Reply:
@JoeOster Should I squash them into my original commit?

Reply:
they should already be in your commit